### ISSUE TITLE
About config files

### ISSUE THREAD

**Issue by xjixzz** [sunnyHelen/JPerceiver#5]
Hi, 

I am confused that which config file I should choose to obtain the [pretrained models](https://drive.google.com/drive/u/0/folders/1QAUEluGvSimXsvtvpXE0FxMR7jCxm0BO). Could you please provide scripts for training and evaluating [these models](https://drive.google.com/drive/u/0/folders/1QAUEluGvSimXsvtvpXE0FxMR7jCxm0BO)?

Besides, I have some questions about the setups in the configuration file:
(1) what is the gt depth file? Does it correspond to the validation set of kitti odometry? 
https://github.com/sunnyHelen/JPerceiver/blob/22ea4c8d2350b54c1b0c45b45466404acc556fd0/config/cfg_kitti_baseline_odometry_boundary_ce_iou_1024_20.py#L16

（2）Although  loss_sum is set to 3, loss_weightS is missing in the configuration file.
https://github.com/sunnyHelen/JPerceiver/blob/22ea4c8d2350b54c1b0c45b45466404acc556fd0/config/cfg_kitti_baseline_odometry_boundary_ce_iou_1024_20.py#L52
https://github.com/sunnyHelen/JPerceiver/blob/22ea4c8d2350b54c1b0c45b45466404acc556fd0/mono/model/mono_baseline/net.py#L581-L582

**Comment by sunnyHelen** [comment]
Sorry for the late reply. 
(1) the gt_depths.nps is for the ground truth depth for the Eigen split. It doesn't correspond to the validation set of kitti odometry.
(2) the opt.loss_weightS = 20
The config files are a little messy. I will sort it out later and modify the readme file.
